.buildkite/scripts/test_synapse_port_db.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -xe
```

Comment:
`set -e` is more conventional as a separate command: it guards against people running the script as `bash <script>` without the -e.